src/components/EditorPage.js

```diff
@@ -173,7 +174,7 @@ class EditorPage extends Component {
             icon='ion-android-clipboard'
             title='Copy'>
 
-      <CopyToClipboard text={this.props.template.get('html')}>
+            <CopyToClipboard text={beautify.html(this.props.template.get('html'))}>
```

Comment:
I'm not a big fan of calling `beautify.html` in the `render` method, it means everytime whatever happen, it will call `beautiful.html` which can be pretty costful in term of perf. I think we can avoid calling it too often by adding, on a `componentWillReceiveProps`:

```javascript
componentWillReceiveProps (nextProps) {
  const curHTML = this.props.template.get('html')
  const nextHTML = nextProps.template.get('html')
  if (curHTML !== nextHTML) {
    this._beautifiedHTML = beautify.html(nextHTML)
  }
}
```

... and use `this._beautifiedHTML` in render (it also needs to be instancied in `componentWillMount`.